src/Error/Debugger.php

```diff
@@ -32,6 +33,16 @@
  */
 class Debugger
 {
+    use InstanceConfigTrait;
```

Comment:
I understand what you're saying. It was a little difficult to figure out how to approach this feature. If you want it done a different way, just let me know.